package.json

```diff
@@ -86,12 +86,12 @@
   "devDependencies": {
     "@appfolio/eslint-config-appfolio-react": "^2.0.2",
     "@babel/cli": "^7.17.6",
-    "@babel/core": "^7.17.8",
-    "@babel/preset-env": "^7.16.11",
+    "@babel/core": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
     "@babel/preset-react": "^7.16.7",
-    "@babel/preset-typescript": "^7.16.7",
+    "@babel/preset-typescript": "^7.24.7",
     "@chromatic-com/storybook": "^1.5.0",
-    "@jest/types": "^27.5.1",
+    "@jest/types": "^29.6.3",
```

Comment:
Shouldn't this be v28?

Reply:
no i updated, jest to v29

Reply:
Ah, I got confused with `@types/jest`. Can you upgrade that to v29 as well. It seems like it's available https://www.npmjs.com/package/@types/jest

Reply:
yes, thanks!